src/Roave/InfectionStaticAnalysis/Psalm/RunStaticAnalysisAgainstMutant.php

```diff
@@ -5,26 +5,36 @@
 namespace Roave\InfectionStaticAnalysis\Psalm;
 
 use Infection\Mutant\Mutant;
+use Psalm\Internal\Analyzer\IssueData;
 use Psalm\Internal\Analyzer\ProjectAnalyzer;
 
-use function array_key_exists;
+use function array_map;
 use function count;
+use function implode;
 
 /**
  * @internal
+ * @psalm-suppress InternalProperty - we use Psalm's internal IssueData class here. Afaik the only other way to
+ * display details of the issues would be to use one of the subclasses of \Psalm\Report, but I think none are exactly
+ * what we want. Probably we can accept the risk of Psalm's internals changing and breaking this.
```

Comment:
I'm also happy to have a go at using one of the subclasses of https://github.com/vimeo/psalm/blob/5.x/src/Psalm/Report.php to avoid the InternalProperty access.

Reply:
Note: we pin to the patch version of `infection/infection` because we want to prevent inadverted BC breaks.

If we can avoid depending on `@internal` in Psalm, we could avoid having to do it for `vimeo/psalm`